src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FiguresSupplier figureSupplier = new FiguresSupplier();
+
+        for (int i = 0; i < 3; i++) {
+            figureSupplier.getRandomFigure().draw();
+        }
+        for (int i = 0; i < 3; i++) {
```

Comment:
The loop should iterate over half of the total number of figures you want to generate. Consider using a variable to define the total number of figures and then use it to calculate the number of random and default figures.